docs/faq.md

```diff
@@ -54,3 +54,10 @@ https://api.refine.bio/processors/
 ```
 
 In addition, you may wish to obtain <a href ="https://hub.docker.com/u/ccdl/" target = "blank">our Docker images</a> (prefixed with `dr_`) which will allow you to access version information for every dependency.
+
+#### Are refine.bio datasets I download batch corrected?
+
+refine.bio datasets are **not** processed with tools that perform batch correction by altering the gene expression values such as ComBat ([Johnson et al. _Biostatistics_. 2007.](https://doi.org/10.1093/biostatistics/kxj037)).
```

Comment:
I might recommend re-ordering to something like: 

```suggestion
We apply quantile normalization to mitigate issues caused by differences in the underlying distributions of gene expression values in samples.
This makes the gene expression values broadly comparable, but doesn't explicitly correct for batch, dataset, or platform.
If the scientific question and analysis methods require datasets to be batch corrected, users should first investigate the existence of batch effects using methods such as Principal Components Analysis.
If the source dataset is associated with major sources of variability in the data, users may wish to use a meta-analysis framework considering each dataset independently or to apply a batch correction tool..
For certain analyses it may be sufficient to include batch, dataset, or platform as covariates.
```